dependency discrepancy

Not quite sure what's going on here but when I run `npm install spawn-sync@1.0.6` I get a package.json which has a single dependency declared:
```
{
  "name": "spawn-sync",
  "version": "1.0.6",
  "dependencies": {
    "concat-stream": "^1.4.7"
  }
}
```

However, its node_modules folder contains the following two dependencies:
- concat-stream: 1.4.8
- os-shim: 0.1.2

This conflict causes `npm shrinkwrap` and `npm ls` to rightfully fail and complain about extraneous modules:
```
npm ERR! extraneous: os-shim@0.1.2 ./node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-wrapper/node_modules/bin-check/node_modules/spawn-sync/node_modules/os-shim
```

Running `npm info spawn-sync@1.0.6` on the other hand has the extra dependency listed:
```
dependencies: { 'concat-stream': '^1.4.7', 'os-shim': '^0.1.2' }
```

Has this version been published correctly or is this a local problem with my npm?

--- a/postinstall.js
+++ b/postinstall.js
@@ -5,10 +5,12 @@ var cp = require('child_process');
 var assert = require('assert');
 
 var partialDependencies = {
-  "concat-stream": "^1.4.7"
+  "concat-stream": "^1.4.7",
+  "os-shim": "^0.1.2"
 };
 var fullDependencies = {
   "concat-stream": "^1.4.7",
+  "os-shim": "^0.1.2",
   "try-thread-sleep": "^1.0.0"
 };
 

